frontend/src/components/pages/admin/UsersPage.tsx

```diff
@@ -194,7 +194,6 @@ const UsersPage = (): React.ReactElement => {
     }
   };
 
-  type Role = "teacher" | "admin";
   return (
     <>
       <Box>
```

Comment:
Linting

Reply:
i think this should disappear once you pull from staging